task/pages/_document.tsx

```diff
@@ -1,13 +0,0 @@
-import { Html, Head, Main, NextScript } from 'next/document'
-
-export default function Document() {
-  return (
-    <Html lang="en">
-      <Head />
-      <body>
-        <Main />
-        <NextScript />
-      </body>
-    </Html>
-  )
-}
```

Comment:
We are deleting `_document.tsx`, but not creating a `_document.js`. Does this work?